playfab.json

```
@@ -3,6 +3,57 @@
     "name": "PlayFab Postman SDK 0.110.200914",
     "description": "PlayFab API SDK for Postman.\n\nThis is still under development, and is not yet ready for general use.  Experienced users can utilize this if they carefully examine the post-body and ensure the data is properly entered.  By default, the post-body is NOT defaulting to useable values.",
     "order": [],
+	"events": [
+		{
+			"listen": "prerequest",
+			"script": {
+				"id": "b7114d52-8fee-497d-bb42-4b4d1446174c",
+				"type": "text/javascript",
+				"exec": [
+					"// When enabled, the following script will ensure that the EntityToken environment",
+					"// variable always contains a valid title entity token. To enable the script, create",
+					"// a global variable named PlayFab_AutoTitleToken and set its value to true.",
+					"",
+					"var autoTitleToken = pm.globals.get(\"PlayFab_AutoTitleToken\");",
```

I think I'd rather the toggle be per-environment rather than a global.
```suggestion
					"var autoTitleToken = pm.environment.get(\"PlayFab_AutoTitleToken\");",
```

---

I liked your suggestion to allow either a global or environment variable.